cross/he853/Makefile

```diff
@@ -20,4 +20,4 @@ include ../../mk/spksrc.cross-cc.mk
 .PHONY: he853_install
 he853_install:
 	mkdir -p $(STAGING_INSTALL_PREFIX)/lib/udev/rules.d
-	$(RUN) $(MAKE) install DESTDIR=$(INSTALL_DIR) PREFIX=$(INSTALL_PREFIX) MY_UDEV_RULES=$(STAGING_INSTALL_PREFIX)/lib/udev/rules.d
+	$(RUN) $(MAKE) install PREFIX=$(INSTALL_DIR)$(INSTALL_PREFIX) MY_UDEV_RULES=$(STAGING_INSTALL_PREFIX)/lib/udev/rules.d
```

Comment:
This is not relevant to me. Please revert or explain what is wrong when using both DESTDIR and PREFIX. From my point of view, everything is OK.

Reply:
without this fix, the installation goes to $(PREFIX) (/var/packages) instead of $(INSTALL_DIR)$(PREFIX). i.e. the DESTDIR variable is not evaluated for make install

Reply:
@ymartin59 is there anything I can do for this?